submissions/Yasya23/holi-style-popup/style.css

```diff
@@ -0,0 +1,177 @@
+* {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+img {
+  display: block;
+}
+
+:focus {
+  outline: none;
+}
+
+.navigation {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  margin: 0.3rem 0.6rem 0.6rem 0.3rem;
+}
+
+.navigation__list {
+  position: relative;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: right;
+  -ms-flex-pack: right;
+  justify-content: right;
+  margin: 1rem;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.navigation__item,
+.navigation__popup-item {
+  margin-right: 0.6rem;
+  padding: 0.3rem;
+}
+
+.navigation__popup-icon {
+  max-width: 2.5rem;
+  height: 100%;
+  cursor: pointer;
+}
+
+.popup__input,
+.popup__more-input {
+  position: absolute;
+  right: 5rem;
+  top: 0.7rem;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
+}
+
+.popup__container {
+  display: none;
+  position: absolute;
+  top: 3.1rem;
+  right: -0.2rem;
+  width: 20rem;
+  max-height: 30rem;
+  -webkit-box-shadow: 0 1px 6px 0 rgba(32, 33, 36, 0.278);
+  box-shadow: 0 1px 6px 0 rgba(32, 33, 36, 0.278);
+  border-radius: 5px;
+  overflow: auto;
+}
+
+.popup__list {
+  display: grid;
+  grid-template: auto/1fr 1fr 1fr;
+  gap: 0.6rem;
+  padding: 1.2rem;
+}
+
+.popup__box-img {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  overflow: hidden;
+  padding: 0.3rem;
+}
+
+.popup__box-img:hover,
+.navigation__item:hover {
+  border-radius: 5px;
+  background-color: #d9f2ff;
+  cursor: pointer;
+}
+
+.popup__box-img:focus-within,
+.navigation__item:focus-within {
+  outline: 2px solid #6395ea;
+  border-radius: 5px;
+}
+
+.popup__img {
+  max-width: 100%;
```

Comment:
is properties ` max-width` and `height` are needed?

Reply:
Fixed it